using-mobile-extensions/adobe-target/README.md

```diff
@@ -200,30 +255,55 @@ TargetOrder targetOrder = new TargetOrder("123", 567.89, purchasedProductIds);
 ```
 {% endtab %}
 
-{% tab title="iOS" %}
+{% tab title="iOS (AEP 3.x)" %}
 #### Syntax
 
-```objectivec
-+ (nonnull instancetype) targetOrderWithId: (nonnull NSString*) orderId
-total: (nullable NSNumber*) total
-purchasedProductIds: (nullable NSArray <NSString*>*) purchasedProductIds;
+```swift
+@objc public init(id: String, total: Double = 0, purchasedProductIds: [String]? = nil)
```

Comment:
@calebk1 We only have Swift syntax for the AEP version of SDK. The Swift SDK code can be accessed from both Swift and Objective C applications. So examples contain both Swift and ObjectiveC.

Reply:
@shtomar-adb Can we backport the Objective-C syntax? Since, at least to me, it feels like a poor user experience to have both Objective-C and Swift examples, but only give the syntax of Swift.

Reply:
@calebk1 sure we can do that. I followed other extension that have added separate tab for AEP 3.x and ACP 2.x for ex: https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-analytics-mobile-services#set-up-push-messaging.
Do you think arranging tab like this https://aep-sdks.gitbook.io/docs/foundation-extensions/places/migration#getcurrentpointsofinterest is better?

Reply:
After some mulling, I think arranging it so we have the second way is better - that way, users know automatically what version and language they are looking for.

Sorry to make you change it like this!

Reply:
I think we need consistency here. Last I knew we were following this formatting https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/pull/680#discussion_r772664977 as demonstrated here https://aep-sdks.gitbook.io/docs/foundation-extensions/identity-for-edge-network/api-reference#getexperiencecloudid